src/Http/Headers/src/ContentDispositionHeaderValue.cs

```diff
@@ -183,11 +183,11 @@ public StringSegment FileNameStar
             }
             else if (sizeParameter != null)
             {
-                sizeParameter.Value = value.GetValueOrDefault().ToString(CultureInfo.InvariantCulture);
+                sizeParameter.Value = value.Value.ToString(CultureInfo.InvariantCulture);
```

Comment:
I don't think we should be making these changes. The analyzer has a bug in it. We should be suppressing the false positive warnings until the bug is fixed.

Reply:
I was going to file a bug too. I figured this makes the close marginally better by making it more self-documenting that we don't expect null. I like it better than suppressions that we'll have to undo.

Reply:
I (re)opened the analyzer bug at https://github.com/dotnet/roslyn-analyzers/issues/6746